server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -669,6 +671,18 @@ protected Node(
                 clusterService.getClusterSettings(),
                 threadPool::relativeTimeInMillis
             );
+            final SetOnce<RemoteClusterStateService> remoteClusterStateServiceReference = new SetOnce<>();
```

Comment:
do we really need SetOnce here?

Reply:
This is not required. Was looking to use a function reference during binding. Will instead use anonymous function